imageprocessor/ocr_test.go

```diff
@@ -25,20 +25,20 @@ func TestStandardOCR(t *testing.T) {
 	}
 }
 
-func TestDuelOCR(t *testing.T) {
-	image, err := getUploadedFileObject()
-	if err != nil {
-		t.Fatalf("Could not initialize standard OCR test")
-	}
-	defer image.Clean()
-
-	ocrStratagy := DuelOCRStratagy()
-	ocrStratagy.Process(image)
-
-	if image.GetOCRText() != "hello" {
-		t.Fatalf("Did not get proper Duel OCR text back %s != hello", image.GetOCRText())
-	}
-}
+// func TestDuelOCR(t *testing.T) {
+// 	image, err := getUploadedFileObject()
+// 	if err != nil {
+// 		t.Fatalf("Could not initialize standard OCR test")
+// 	}
+// 	defer image.Clean()
+//
+// 	ocrStratagy := DuelOCRStratagy()
+// 	ocrStratagy.Process(image)
+//
+// 	if image.GetOCRText() != "hello" {
+// 		t.Fatalf("Did not get proper Duel OCR text back %s != hello", image.GetOCRText())
+// 	}
+// }
```

Comment:
Lets go ahead and remove. We can restore from git if we decide to bring this feature back